supply-chain/config.toml

```diff
@@ -1299,6 +1291,10 @@ criteria = "safe-to-deploy"
 version = "0.26.0"
 criteria = "safe-to-deploy"
 
+[[exemptions.quick_cache]]
+version = "0.6.0"
```

Comment:
```suggestion
version = "0.6.2"
```
The version noted in `cargo vet` should be the same version that you have in `Cargo.lock`.